pr-Template.md

```diff
@@ -0,0 +1,16 @@
+﻿
```

Comment:
Please remove this file from your PR. We are looking for this info in the PR template which you already filled out here

![image](https://user-images.githubusercontent.com/28658112/179048104-06daccad-b4c4-4957-9a8d-8baf74c37f03.png)

Reply:
Has been deleted.